pkg/sloop/ingress/kubewatcher.go

```diff
@@ -122,7 +122,6 @@ func (i *kubeWatcherImpl) startWellKnownInformers(kubeclient kubernetes.Interfac
 	i.informerFactory.Core().V1().PersistentVolumeClaims().Informer().AddEventHandler(i.getEventHandlerForResource("PersistentVolumeClaim", enableGranularMetrics))
 	i.informerFactory.Core().V1().PersistentVolumes().Informer().AddEventHandler(i.getEventHandlerForResource("PersistentVolume", enableGranularMetrics))
 	i.informerFactory.Core().V1().Pods().Informer().AddEventHandler(i.getEventHandlerForResource("Pod", enableGranularMetrics))
-	i.informerFactory.Policy().V1().PodDisruptionBudgets().Informer().AddEventHandler(i.getEventHandlerForResource("PodDisruptionBudget", enableGranularMetrics))
```

Comment:
🤔 The error report said sloop was polling for pdbs in `v1beta1`, but here we're looking at `v1` which looks right based on the [pdb docs](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).

Reply:
Ah, the screenshot shows the call was made on August 30, and Jimi updated PDB checks to v1 in https://github.com/mitchnielsen/sloop/pull/2.

Reply:
And then in https://github.com/PrefectHQ/platform/pull/7346, he updated the reference in the Platform project to point to that new version of sloop. I believe we're just using the `latest` image tag for sloop at the moment, so it propogated in the clusters some time after that.

I bet we won't see this error anymore and can clear it from the cloud console (if possible).

Reply:
ah i see, thats my bad! I got this confused with something other resource that is deprecated completely. ignore me :)